src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -13,18 +13,19 @@
 ]
 
 
-def write_caption_data_to_file(caption_data: dict):
+def write_caption_data_to_file(caption_data: dict, file_name: str):
     """Writes caption data to file as json.
 
        :param caption_data: json with byte pairs
+       :param file_name: used for saving resulting file
     """
     # datetime object containing current date and time
     now = datetime.now()
     # mm.dd.YY_H:M:S
     dt_string = now.strftime("%m.%d.%Y_%H-%M-%S")
 
     path = config.path_to_data_folder
-    file_name = f'output_{dt_string}.json'
+    file_name = file_name + f'output_{dt_string}.json'
```

Comment:
Use time_stamp here, described in previous comment.

Reply:
Done.